src/composition/run_composition.rs

```diff
@@ -143,8 +143,9 @@ mod tests {
             .read_file(mock_read_file)
             .build();
 
-        let subgraph_change_events: BoxStream<SubgraphChanged> =
-            once(async { SubgraphChanged::default() }).boxed();
+        let subgraph_change_events: BoxStream<SubgraphEvent> =
```

Comment:
I'm not sure we even need this/want this? I'm not sure what the value is of producing an empty event.